teacher.rb

```diff
@@ -0,0 +1,10 @@
+module Teacher
+  def initialize(specialization)
+    @specialization = specialization
+    super
```

Comment:
- [ ] Kindly update the teacher class also as asked by the previous reviewer. Initialize `age`, `name = "Unknown"` and `parent_permission = true` and add its super classes